servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -110,8 +110,11 @@ quarkus.resteasy.gzip.enabled=true
 #quarkus.oidc.auth-server-url=
 # fixed at buildtime
 quarkus.http.auth.basic=false
-quarkus.oidc.enabled=false
-
+quarkus.oidc.enabled=true
+quarkus.oidc.client-id=projectnessie
+nessie.auth.enabled=false
+# to be overwritten by end user when enabling OpenID validation
+quarkus.oidc.auth-server-url=http://localhost/auth/realms/unset/
```

Comment:
Is this parameter mandatory if `quarkus.oidc.enabled=true` is set?

Reply:
It has to be syntactically valid, otherwise the server will not start. It will not be used unless `quarkus.oidc.enabled` is `true`.

Reply:
I changed the default URL to use a documentation IP address in 089e6c36d10a943566eff5e094a161b4653da443